components/transfer/TransferConfirmModal.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <NeoModal v-model="isModalActive" scroll="clip" @close="emit('close')">
+    <div
+      class="modal-width is-flex is-flex-direction-column is-justify-content-space-between">
+      <div>
+        <div
+          class="is-flex is-justify-content-center is-align-items-center header px-6 py-3">
+          <span class="is-size-5 has-text-weight-bold">
+            {{ $t('teleport.send') }}
+            <span class="is-uppercase">{{ unit }}</span>
+          </span>
+          <NeoIcon
+            class="position-right is-clickable mr-6"
+            icon="close"
+            pack="fas"
+            @click.native="closeModal" />
```

Comment:
please check `ConfirmPurchaseModal` component

Reply:
I Updated it. How about now? There is something structure different between them.

Reply:
> I Updated it. How about now? There is something structure different between them.

We shouldn't, should be unified design

Reply:
Emm, but they are not unified now from design.

TransferConfirmModal:
<img width="311" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/07ebda70-1da9-4925-b0c7-48f61b78d9c0">


ConfirmPurchaseModal:
<img width="443" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/eb35bff7-dc7e-400a-b67d-5304e9afc488">

Reply:
@kodadot/design-squad wdyt

Reply:
The designs usually differ here so the ideal situation would be consistency...

If there is already a header component, it would be better to use that instead of creating a new one.